Hide notification when a user forks a repo

paulmolluzzo opened the pull request:
This fixes #113 by hiding the notification when someone forks another repo. :v:

hkdobrev commented:
:+1: 

@paulmolluzzo Do you think you could unify all feed item `display: none` rules? We could just have a list of selectors with a single comment, consistent style of selectors and CSS properties.

paulmolluzzo commented:
OK, I pushed up a change that combined them all. Let me know if I messed up when moving around pieces.

hkdobrev commented on extension/content.css:
```diff
@@ -213,26 +213,21 @@
+create a branch
+deletes a branch
+adds someone as a collaborator
+forks a repo
```
Could we use consistent present imperative wording?
```
push to a branch
create a branch
delete a branch
add someone as a collaborator
fork a repo
```

paulmolluzzo replied:
LOL, don't mean to argue, but they should all be past tense. They happened already, so we got a notification. :wink:

Also going to add `!important` a la https://github.com/sindresorhus/refined-github/pull/22#discussion_r53374626

paulmolluzzo commented:
All past tense:
```
pushed to a branch
created a branch
deleted a branch
added someone as a collaborator
forked a repo
```

Using `!important`.

hkdobrev commented:
LGTM from me. I don't want to get into more details when the goal is to first fix #113.
